### PR TITLE
Upgrade to SARIF SDK version 2.2.5

### LINKED ISSUE #141: Viewer can't open SARIF functional test file
If you open the SARIF SDK functional test file https://github.com/microsoft/sarif-sdk/blob/master/src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1017.InvalidIndex_Invalid.sarif in the VS Extension, it throws the error "Value cannot be null. Parameter name: key".

### PR DESCRIPTION
**Pull request by eddynaka**
Changes:
- updating sarif packages from `src/Sarif.Viewer.VisualStudio.UnitTests/Sarif.Viewer.VisualStudio.UnitTests.csproj`
- updating sarif packages from `src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj`
- updating test to prevent null reference exception
- updated `SuprresionState` to `SuppressionStatus`
- added to release notes 2.1.9

### CODE REVIEW

File: src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ internal static List<CallTreeNode> Convert(CodeFlow codeFlow, Run run)
 
                 foreach (ThreadFlowLocation location in threadFlow.Locations)
                 {
-                    ArtifactLocation artifactLocation = location.Location.PhysicalLocation?.ArtifactLocation;
+                    ArtifactLocation artifactLocation = location.Location?.PhysicalLocation?.ArtifactLocation;
```
Comment (lgolding) [inline src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs:26]: >on?.Phy [](start = 71, length = 7)

I assume this was the actual bug fix.

Comment (eddynaka) [reply src/Sarif.Viewer.VisualStudio/CodeFlowToTreeConverter.cs:26]: I changed this after updating. So, when I started to check the unit test, they were failing. And with that, they started to pass.

**Comment by eddynaka** [comment]
@lgolding , below some info:

*master branch*:
- 97 tests ok
- 24 tests not ok

*fixing-viewer-sarif branch* (_without_ nuget update):
- 113 tests ok
- 8 test not ok

*fixing-viewer-sarif branch* (_with_ nuget update):
- 121 tests ok

**Comment by eddynaka** [comment]
@lgolding , about the version in manifest, should i just increment to 2.1.10 and add in the releaseHistory.md? Thank you.

**Pull request merged**